development/image-syncer/external-images.yaml

```
@@ -12,8 +12,8 @@ images:
 - source: "gcr.io/google-containers/pause:3.2"
 - source: "goreleaser/goreleaser:v1.11.5"
 # Golang image is pinned because it is force updated in the upstream
-- source: "golang@sha256:a9b24b67dc83b3383d22a14941c2b2b2ca6a103d805cac6820fd1355943beaf1"
-  tag: "1.19.4-alpine3.17-v1"
+- source: "golang@sha256:3183b343d01a6e8c9bee7b7c4eb7208518e68dc0cdac8623e1575820342472ed"
+  tag: "1.19.5-alpine3.17-v1"
```

Now we're using the SHA of the multiplatform manifest: https://hub.docker.com/layers/library/golang/1.19.4-alpine3.17/images/sha256?context=explore
```suggestion
- source: "golang@sha256:86d32cc0dfc04757fd8aeebb86308e6d1e3de60c73cb59e0f99c7b2ef77416b6"
  tag: "1.19.5-alpine3.17"
```

---

Thank you. I cannot push to this branch any more, so I will create a new PR.